_data/changelogs/component-form.yml

```diff
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added guidance discouraging most uses of disabled form inputs.
+    summaryAdditional: We also added guidance on the rare instances when disabled inputs improve usability. 
+    affectsAccessibility: true
+    affectsGuidance: true
+    githubPr: 2776
+    githubRepo: uswds
+    versionUswds: 3.8.2
```

Comment:
**issue (blocking):** The related issue is in the uswds-site repo and is not attached to a specific version of USWDS

```suggestion
    githubRepo: uswds-site
```